action.yml

```diff
@@ -32,6 +32,9 @@ inputs:
     description: "Also cache on workflow failures"
     default: "true"
     required: false
+  key:
```

Comment:
This should be `cache-key` instead of `key` to make it clear that this is a cache related setting.

Reply:
Cheers, [fixed](https://github.com/actions-rust-lang/setup-rust-toolchain/compare/d4e530262382bb10ab572c928e42bf70b5bec312..b01657d9bb1c81c4e7a46604980d19112a4034f0).